ui/src/experiments/ExperimentsLandingPage.js

```diff
@@ -21,62 +21,57 @@ import TreatmentDetailsView from "treatments/details/TreatmentDetailsView";
 import TreatmentHistoryDetailsView from "treatments/history/details/TreatmentHistoryDetailsView";
 import ListTreatmentsView from "treatments/list/ListTreatmentsView";
 
-const ExperimentsLandingPage = ({ projectId }) => {
+const ExperimentsLandingPage = () => {
+  const location = useLocation();
   /* Application Routes should be defined here, as ExperimentsLandingPage component is
      being exposed for use via MFE architecture. */
   return (
     <ConfigProvider>
       <ProjectContextProvider>
-        <Router>
-          <CreateSettingsView path="/settings/create" />
-          <SettingsDetailsView path="/settings/*" />
+        <Routes>
+          {/* SETTINGS */}
+          <Route path="settings">
+            <Route index path="*" element={<SettingsDetailsView />} />
+            <Route path="create" element={<CreateSettingsView />} />
+          </Route>
 
-          <TreatmentHistoryDetailsView path="/treatments/:treatmentId/history/:version" />
-          <LandingView
-            Component={ListTreatmentsView}
-            name="Treatments"
-            projectId={projectId}
-            path="/treatments"
-          />
-          <CreateTreatmentView path="/treatments/create" />
-          <TreatmentDetailsView path="/treatments/:treatmentId/*" />
+          {/* TREATMENTS */}
+          <Route path="treatments">
+            <Route index element={<LandingView Component={ListTreatmentsView} name="Treatments" />} />
+            <Route path="create" element={<CreateTreatmentView />} />
+            <Route path=":treatmentId">
+              <Route path="history/:version" element={<TreatmentHistoryDetailsView />} />
+              <Route index path="*" element={<TreatmentDetailsView />} />
```

Comment:
I switched up the order of the index route. We need the `path="*"` because each component has a nested set of routes under it (eg: The treatment detail view has different tabs).

Reply:
Ahh okay I see, alright then! :)